bin/satellite-clone.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from optparse import OptionParser
```

Comment:
`optparse` is [deprecated in 2.7](https://docs.python.org/2/library/optparse.html), and one should use [`argparse`](https://docs.python.org/2/library/argparse.html) instead. AFAIK `satellite-clone` is only supported on EL7, so depending on Python 2.7 should be fine?